### ISSUE TITLE
Replace tracked color

### ISSUE THREAD

**Issue by MirkoAlo** [eduardolundgren/tracking.js#224]
Hello!

I Can replace the color tracked whit another color??

Thank you for this amazing library

**Comment by JhonImmanuel** [comment]
Absolutely yes.

use registerColor function. Here is some snippet code.

`tracking.ColorTracker.registerColor('yellow', function(r, g, b) {
    var threshold = 50,
      dx = r - 255,
      dy = g - 255,
      dz = b - 0;

    if ((r - b) >= threshold && (g - b) >= threshold) {
      return true;
    }
    return dx * dx + dy * dy + dz * dz < 10000;
  });`

**Comment by revolunet** [comment]
solved ?

**Comment by MirkoAlo** [comment]
Hi Julien!


No i can't solve it, i tracked the color but i don't know i can replace it whit another color... Do you have an example to show me??


Thank you so much for the support,

Mirko

________________________________
Da: Julien Bouquillon <notifications@github.com>
Inviato: domenica 2 luglio 2017 00:01:33
A: eduardolundgren/tracking.js
Cc: MirkoAlo; Author
Oggetto: Re: [eduardolundgren/tracking.js] Replace tracked color (#224)


solved ?

**Comment by JhonImmanuel** [comment]
Did you mean replacing color, let say color in some part in your image?

**Comment by MirkoAlo** [comment]
Yes, I'd like that the background of an image, black for example,  begin White...
________________________________
Da: NPE <notifications@github.com>
Inviato: domenica 2 luglio 2017 18:09:46
A: eduardolundgren/tracking.js
Cc: MirkoAlo; Author
Oggetto: Re: [eduardolundgren/tracking.js] Replace tracked color (#224)


Did you mean replacing color, let say color in some part in your image?

**Comment by cotejp** [comment]
@JhonImmanuel Would you care to elaborate? I'm trying to add your `registerColor()` snippet but it does not work. In fact, I've had no success at all with `registerColor()` so far. 

I'm using it with a webcam feed and, as far as I can tell, the function registered with `registerColor()` is not even being called... Does it need to be called in a specific order? 

Do you perhaps have a full example of using `registerColor()` with a webcam somewhere? There must be something I'm doing wrong but I have no clue what...

**Comment by cotejp** [comment]
Ok, I got it working. The documentation is kinda vague as to the fact that you first need to register the color to a name and then create a new `ColorTracker` using that name as a parameter...

If I understand correctly, you cannot dynamically change the color being tracked. You have to register the new color and create a new `ColorTracker` object, is that right?
